tests/db/pg_client.py

```diff
@@ -21,7 +19,9 @@ def __init__(self, connection_type: ConnectionType) -> None:
         if self.route == ConnectionType.LOCAL:
             self.conn, self.cur = connect_and_populate_db()
         elif self.route == ConnectionType.REMOTE:
-            self.dune = DuneAPI.new_from_environment()
+            # TODO - Change DB setup to use Hive instead of postgres.
+            #  I don't believe this code path is currently used
+            NotImplemented
```

Comment:
```suggestion
            # TODO - Change DB setup to use Hive instead of postgres.
            #  I don't believe this code path is currently used
            # https://github.com/cowprotocol/solver-rewards/issues/147
            NotImplemented
```